packages/liveblocks-node/src/utils.ts

```diff
@@ -36,9 +47,15 @@ export function assertSecretKey(
   value: unknown,
   field: string
 ): asserts value is string {
-  if (!isNonEmpty(value) || !value.startsWith("sk_")) {
+  if (!startsWith(value, "sk_")) {
```

Comment:
NB: Shouldn't we strengthen the condition with:
```ts
if (!startsWith(value, "sk_dev_") && !startsWith(value, "sk_prod_")) {
  // ....
}
```

Because if I'm not wrong as it's written it will pass with things like `sk_something_.....` no? Or do am I overthinking here?

Reply:
The `sk_{dev,prod}` pattern is relatively new. I believe there may still exist older `sk_` values that won't have that format unless those keys would be rolled.

> Because if I'm not wrong as it's written it will pass with things like `sk_something_.....` no? Or do am I overthinking here?

In those cases, the key would be syntactically correct, but just not work because it's not a valid value. I think we're good in that case. The point of these checks is to avoid misconfigurations and provide guidance, not to catch all edge cases that aren't valid _today_. Adding a check like `sk_{dev,prod}` would prevent us from changing the key format in the future, so the check is deliberately somewhat loose.

Reply:
Oh okay! I wasn't aware of the previous key pattern. Thanks for sharing 🙏🏻 
Yes I agree then in those cases the check is good enough as it is 👍🏻